lib/features/membership_feature/widgets/member_list_item_widget.dart

```diff
@@ -0,0 +1,67 @@
+import 'package:FitnessPassport/components/label_widget.dart';
+import 'package:FitnessPassport/components/profile_image_widget.dart';
+import 'package:FitnessPassport/components/right_button_widget.dart';
+import 'package:FitnessPassport/utils/themes/colors.dart';
+import 'package:FitnessPassport/utils/themes/text_styles.dart';
+import 'package:flutter/material.dart';
+
+class MemberListItem extends StatelessWidget {
+  final String? memberName;
+  final bool? isMemberPrimary;
+  const MemberListItem({super.key, this.memberName, this.isMemberPrimary});
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: const EdgeInsets.symmetric(horizontal: 15.0, vertical: 10.0),
+      decoration: BoxDecoration(
+        color:
+            isMemberPrimary! ? CustomColors.primaryBlue : CustomColors.white01,
+        borderRadius: const BorderRadius.all(Radius.circular(16.0)),
+      ),
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        children: [
+          Row(
+            children: [
+              const ProfileImage(),
+              const SizedBox(
+                width: 10.0,
+              ),
+              Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  Label(
+                    text: memberName,
+                    fontStyle: tHeader2,
+                    textColor: isMemberPrimary!
+                        ? CustomColors.white01
+                        : CustomColors.greyDark,
+                  ),
+                  Label(
+                    text: isMemberPrimary!
+                        ? 'Primary Member'
+                        : 'Secondary Member',
+                    fontStyle: tSubTitle2,
+                    textColor: isMemberPrimary!
+                        ? CustomColors.white01
+                        : CustomColors.greyDark,
+                  ),
+                ],
+              ),
+            ],
+          ),
+          RightButtonWidget(
```

Comment:
we need to make the entire list item clickable, rather than concentrating the click option to this small option.

Reply:
Code updated with that requirement